src/server.h

```diff
@@ -976,8 +985,10 @@ typedef struct redisDb {
     int id;                     /* Database ID */
     long long avg_ttl;          /* Average TTL, just for stats */
     unsigned long expires_cursor; /* Cursor of the active expire cycle. */
+    int resize_cursor;          /* Cron job uses this cursor to gradually resize dictionaries. */
     list *defrag_later;         /* List of key names to attempt to defrag one by one, gradually. */
-    clusterSlotToKeyMapping *slots_to_keys; /* Array of slots to keys. Only used in cluster mode (db 0). */
+    int dict_count;             /* Indicates total number of dictionaires owned by this DB, 1 dict per slot in cluster mode. */
+    dbState db_type[DICT_TYPE_MAX];  /* Metadata for main and expires dictionaries */
```

Comment:
```suggestion
    dbState sub_dict[DICT_TYPE_MAX];  /* Metadata for main and expires dictionaries */
```
db_type implies this is the type of db. This is more like a subordinate dictionary. Don't really like the name though, and am open to alternatives.